package.json

```diff
@@ -16,12 +16,13 @@
   },
   "author": "Jesús Espino <jesus.espino@mattermost.com>",
   "dependencies": {
-    "@typescript-eslint/typescript-estree": "5.8.1",
+    "@typescript-eslint/typescript-estree": "5.44.0",
```

Comment:
Would you mind changing the dependencies you updated to be inexact to avoid potential future conflicts? Like this'd be `^5.44.0`